test/unittests/common_query/test_common_query.py

```diff
@@ -0,0 +1,95 @@
+import json
+import unittest
+
+from mycroft.skills.intent_services.commonqa_service import CommonQAService
+from ovos_tskill_fakewiki import FakeWikiSkill
+from ovos_utils.messagebus import FakeBus, Message
+
+
+class TestCommonQuery(unittest.TestCase):
+    def setUp(self):
+        self.bus = FakeBus()
+        self.bus.emitted_msgs = []
+
+        def get_msg(msg):
+            self.bus.emitted_msgs.append(json.loads(msg))
+
+        self.skill = FakeWikiSkill()
+        self.skill._startup(self.bus, "wiki.test")
+
+        self.cc = CommonQAService(self.bus)
+
+        self.bus.on("message", get_msg)
+
+    def test_common_query_events(self):
+        self.bus.emitted_msgs = []
+        self.assertEqual(self.cc.skill_id, "common_query.openvoiceos")
+
+        self.bus.emit(Message("common_query.question",
+                              {"utterance": "what is the speed of light"}))
+
+        expected = [
+            # original query
+            {'context': {},
+             'data': {'utterance': 'what is the speed of light'},
+             'type': 'common_query.question'},
+            # thinking animation
+            {'type': 'enclosure.mouth.think',
```

Comment:
Should this be checked here? Seems like these messages belong more in a PHAL plugin?

Reply:
some enclosure messages are emitted by core, PHAL plugins react to them if wanted, eg. mk1 faceplate or mk2 led animation